config.yaml

```diff
@@ -110,6 +110,11 @@ php: 7.3
 # Maximum file upload size. This will set post_max_size and upload_max_filesize in PHP and client_max_body_size in Nginx.
 upload_size: 1024M
 
+# Vagrant machine name
+# This can be any string and helps to identify a specific VM
+# NOTE: If changed the machine will need to be re-created and should ideally be destroyed before any change
+machine_name: default
```

Comment:
Will this make the machine name statically `default`? The actual default in Vagrant is `basename( directory ) + machine_name + timestamp + some_other_number`

Reply:
Let me double check real quick

Reply:
I thought it was a special value - added if statements to only apply the setting if it's not `default`

Reply:
😗 👌